### DISCUSSION TITLE
RKE2 support?

### DISCUSSION THREAD

**Discussion by 4FunAndProfit** [vitobotta/hetzner-k3s#503] [Q&A]
Hello ! 

I know it is in the title of the repository that it is k3s haha 😝

But was wondering if it is planned to support as well rke2? Would be amazing (for enterprise grade and security minded)

Thanks a lot in advance :)

**Comment by vitobotta** [comment]
Hi, I'm turning this into a discussion since it's not really an issue with the tool.

For the future, I might add support for RKE2, but it's not on the agenda right now because I have a bunch of higher priorities to focus on.
